config/providers.php

```diff
@@ -1,18 +0,0 @@
-<?php
-
-declare(strict_types=1);
-
-use Yiisoft\Di\Container;
-use Yiisoft\Di\Contracts\ServiceProviderInterface;
-use Yiisoft\Widget\WidgetFactory;
-
-return [
-    'yiisoft/widget' => static function (): ServiceProviderInterface {
```

Comment:
That has a drawback. Widgets won't work out of the box anymore. You'll have to initialize it in the application.

Reply:
Fixed